CHANGELOG.D/450.feature

```diff
@@ -0,0 +1 @@
+Avoid too small build presets when none is provided
```

Comment:
if we apply the logic proposed below, we better slightly change it to smth like "Help user to select the resource preset for image build or transfer operations".